controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/util/Result.java

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright 2017 Pinterest, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.pinterest.rocksplicator.controller.util;
+
+/**
+ * A data structure for holding the result from TaskQueue so that
```

Comment:
I haven't tested it but per the doc(http://www.dropwizard.io/0.8.0-rc1/docs/manual/core.html#error-handling) it says `WebApplicationException` will be mapped to a defined `Response` and throwing `Exception` is preferred than returning `Response`. 

I don't have a strong opinion on it though.